base/applications/notepad/dialog.c

```diff
@@ -1117,82 +1117,90 @@ VOID DIALOG_Replace(VOID)
     DIALOG_SearchDialog(ReplaceText);
 }
 
+typedef struct tagGOTO_DATA
+{
+    INT iLine;
+    INT cLines;
```

Comment:
Prefer using UINT for both (see comment below).

Reply:
OK.